apps/web/pages/apps/installation/[[...step]].tsx

```diff
@@ -92,16 +117,15 @@ const OnboardingPage = ({
     [AppOnboardingSteps.EVENT_TYPES_STEP]: {
       getTitle: () => `${t("select_event_types_header")}`,
       getDescription: (appName) => `${t("select_event_types_description", { appName })}`,
-      stepNumber: 2,
+      stepNumber: isConferencing ? 1 : 2,
```

Comment:
If apps that are non-installable on a team skip the `ACCOUNTS_STEP` can we use the `installableOnTeams` prop?